api/base/serializers.py

```diff
@@ -79,6 +79,14 @@ def to_internal_value(self, data):
         return super(TypeField, self).to_internal_value(data)
 
 
+class JSONAPIListField(ser.ListField):
+    def to_internal_value(self, data):
+        if not isinstance(data, list):
+            self.fail('not_a_list', input_type=type(data).__name__)
+
+        return super(JSONAPIListField, self).to_internal_value(data)
```

Comment:
Slightly more specific case of the error check in the parent class (`ser.ListField.to_internal_value`); makes sense. :octopus: :palm_tree: